src/BlocHost-OSWindow-SDL2/BlOSWindowSDL2Host.class.st

```diff
@@ -10,6 +10,23 @@ Class {
 	#category : #'BlocHost-OSWindow-SDL2'
 }
 
+{ #category : #cleanup }
+BlOSWindowSDL2Host class >> cleanUp [
+	"CleanUp Host"
+
+	self universe closeSpaces.
+	self stop.
+	self start.
+
+	"Destroy all SDL windows except the Pharo IDE (assuming its title ends with '.image')"
+	SDL_Window allInstances
```

Comment:
The use of `allInstances` is discouraged in Pharo in general, but it may be okay for this as it's only a development tool, not to run it in production.

Reply:
Yes I agree. I think that "cleanup" is a dev tool.